Maskbook/Scene/Wallet/WalletSend/ViewModel/RecentlyAddressViewModel.swift

```diff
@@ -46,26 +46,64 @@ class RecentlyAddressViewModel: NSObject {
                 return
             }
             items.append(CellItem.address(address: recentlyAddress.address))
-//            guard
-//                let item = ContactManager.getAssignContact(chainId: currentChainId,
-//                                                           xnetworkId: currentNetworkId, address: recentlyAddress.address).first else {
-//                items.append(CellItem.address(address: recentlyAddress.address))
-//                return
-//            }
-//            let displayName = (item.ensName ?? item.name) ?? ""
-//            items.append(CellItem.contact(name: displayName, address: recentlyAddress.address))
         }
         return items
     }
+    
+    func generateRecentAddressCellItem() -> [CellItem] {
+        let currentChainId = Int64(maskUserDefaults.network.chain.rawValue)
+        let currentNetworkId = Int64(maskUserDefaults.network.networkId)
+        let rencentlyAddress = maskUserDefaults.recentlyAddresses.filter {
+            $0.chainId == Int64(maskUserDefaults.network.networkId)
```

Comment:
Why does it use condition `$0.chainId == Int64(maskUserDefaults.network.networkId)`? I don't think chainId and networkId are the same thing.